RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -1598,6 +1604,13 @@
 			path = Responses;
 			sourceTree = "<group>";
 		};
+		A55D5D56282B30EF00FA7623 /* Recovered References */ = {
```

Comment:
unsure what i did to deserve this

Reply:
😂 this is just Xcode telling you it likes you

Reply:
it happened after some merges, i wasn't the smartest with my branching on this 😿

Reply:
You should be able to find this folder in Xcode and manually delete it.